src/addon.cpp

```diff
@@ -66,35 +66,39 @@ class SHA3Hash: public ObjectWrap {
 
 	static
 	NAN_MODULE_INIT(Init) {
-		Local<FunctionTemplate> t = Nan::New<FunctionTemplate>(New);
-		t->SetClassName(Nan::New<String>("SHA3Hash").ToLocalChecked());
-		t->InstanceTemplate()->SetInternalFieldCount(1);
+		Local<Context> context = target->CreationContext();
 
-		Nan::SetPrototypeMethod(t, "update", Update);
-		Nan::SetPrototypeMethod(t, "digest", Digest);
+		Local<String> className = Nan::New<String>("SHA3Hash").ToLocalChecked();
```

Comment:
:memo: **Implementation Note:** Previously, this was being computed and used twice (L70 and L77).